Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -69,8 +70,14 @@ xxx yyy
         public void ProcRunner_FailsOnTimeout()
         {
             // Arrange
+
+            // Previously the script used "TIMEOUT 2" to pause execution in
+            // the spawned process. However, the test was failing on some machines
+            // with the error "Input redirection is not supported, exiting the process immediately."
+            // Pinging a non-existent address with a timeout is an alternative 
+            // that works if the machine is connected to the internet
             string exeName = TestUtils.WriteBatchFileForTest(TestContext,
-@"TIMEOUT 2
+@"@ping 1.1.1.1 -n 1 -w 2000 >nul
```

Comment:
Do a search on the web: you'll find various approaches, this being one. We want something that is available in different versions of the operating system and that provides a timeout mechanism: _ping_ is one such exe, and it doesn't redirect the input stream so we don't get execution errors.
Having said that, I'm not sure it's any more reliable - it's failed once on an AppVeyor build already. I'll watch it over the next few commits and decide whether to revert the change.